packages/dear-imgui/ts/src/lib/components/TradingGuiDemo/CryptoPlots/CryptoCandlestickPlots.tsx

```
@@ -1,15 +1,15 @@
 import React from "react";
 import { useStore } from "../store";
 import { CryptoCandlestickPlot } from "./CryptoCandlestickPlot";
-import { ReactImgui } from "../../ReactImgui";
+import { XFrames } from "../../XFrames";
 
 type Props = {};
 
 export const CryptoCandlestickPlots = ({}: Props) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)